app/controllers/donations_controller.rb

```diff
@@ -38,11 +38,15 @@ def validate_params
 
   def validate_session
     if session[:donations]
-      session[:donations].merge(create_cart) do |key, old_val, new_val|
-        (old_val.to_i + new_val.to_i).to_s
-      end
+      add_to_cart
     else
       create_cart
     end
   end
+
+  def add_to_cart
+   session[:donations].merge(create_cart) do |key, old_val, new_val|
```

Comment:
Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used.<br>Use 2 (not 1) spaces for indentation.